scripts/kubectl-kubestellar-ensure-location

```diff
@@ -159,16 +179,16 @@ while (( index < ${#labels[*]} )); do
     let index=index+2
     stval=$(jq -r <<<"$stlabs" ".[\"${key}\"]")
     if [ "$stval" != "$val" ]; then
-	if ! kubectl "${kubectl_flags[@]}" label --overwrite SyncTarget "$objname" "${key}=${val}" ; then
+	   if ! kubectl "${kubectl_flags[@]}" label --overwrite synctargets.edge.kcp.io "$objname" "${key}=${val}" ; then
```

Comment:
Why is the indentation of this "if" being increased?  Before this change it sat properly between its enclosing indentation and the indentation of the enclosed commands.